commands/initial/templates/protractor.conf.js

```diff
@@ -0,0 +1,39 @@
+const {SpecReporter} = require('jasmine-spec-reporter');
```

Comment:
Just like Webpack & Karma configs, this configuration should be extensible by devs using the library. So if they place a `protractor.conf.js` file in their `configs` directory, this config can be altered.

We'd need to identify the set of attributes the dev could override and which ones angular-librarian should be in charge of.